components/brave_extension/extension/brave_extension/background/docSigner.ts

```diff
@@ -0,0 +1,4 @@
+export const signDocument = () => {
+  const url = "ping://rewards";
+  chrome.tabs.update({ url: url });
```

Comment:
this updates url of current tab only?

Reply:
Yes.. but can open in a new tab as well

Reply:
but which tab is it updating right now. will this not update all tabs

Reply:
No it will only update the tab on which it was called